src/client/pages/Varen/VarenDetailExploitatieHernoemen.tsx

```diff
@@ -44,15 +44,15 @@ export function VarenDetailPageContentExploitatieHernoemen({
       rows: [
         {
           label: 'Lengte vaartuig',
-          content: vergunning.vesselLength
-            ? `${vergunning.vesselLength} meter`
+          content: zaak.vergunning?.vesselLength
+            ? `${zaak.vergunning?.vesselLength} meter`
```

Comment:
Misschien boeit dit niet maar hier weet je al dat er een vesselLength is dus dan hoeft die `?.` ook niet meer.
Een null checkje minder dan toch? Ik zie deze ook verderop weer terugkomen.
```suggestion
            ? `${zaak.vergunning.vesselLength} meter`
```

Reply:
Deze begrijp ik niet? zaak.vergunning kan null zijn want je weet nooit wat je uit decos krijgt. Zie ook Varen.ts

Reply:
de check of de value bestaat doe je al voor `?` daarna weet je dus dat hij bestaat.